core/vm/runtime/runtime.hpp

```diff
@@ -137,9 +137,22 @@ namespace fc::vm::runtime {
     /**
      * @brief Deletes an actor in the state tree
      *
+     * @param address - Address of actor that should be deleted
+     * 
      * May only be called by the actor itself
      */
-    virtual outcome::result<void> deleteActor() = 0;
+    virtual outcome::result<void> deleteActor(const Address &address) = 0;
+
+    /**
+     * @brief Transfer debits money from one account and credits it to another
+     *
+     * @param debitFrom - money sender
+     * @param creditTo - money receiver
+     * @param amount - amount of money that is transfered
+     */
+    virtual outcome::result<void> transfer(const Address &debitFrom,
```

Comment:
Nobody uses `transfer` (except `RuntimeImpl`). Should it be virtual?

Reply:
Is there a possibility that some actor will use it in next versions?